hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -353,6 +359,9 @@ private void handlePlatformTransaction(
             handleOutput.blockRecordSourceOrThrow().forEachItem(blockStreamManager::writeItem);
         }
         opWorkflowMetrics.updateDuration(userTxn.functionality(), (int) (System.nanoTime() - handleStart));
+        if (userTxn.functionality() == TSS_MESSAGE || userTxn.functionality() == TSS_VOTE) {
+            tssMetrics.updateTssTransactionMetrics(userTxn.functionality());
```

Comment:
Actually, I was just contemplating a mechanism for service-specific metrics (see PR #16077, which is injecting metrics for contract service). I'm curious if we can get something like that to work instead, so we don't have to have this in `HandleWorkflow`. It's just an idea at this point, though

Reply:
We won't get to implementing anything like what I had in mind for awhile, so we shouldn't wait for it